Assets/_nathan/scripts/GlobalMusic.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using _nathan.scripts;
+using UnityEngine;
+
+public enum EMusicTrackName
+{
+    MTN_Normal,
+    MTN_Happy,
+    MTN_Intense,
+    MTN_Flourish_Intro,
+    MTN_Flourish_Outro
+}
+
+[Serializable]
+public struct AudioSource_NameSourcePair {
+     public EMusicTrackName name;
+     public AudioClip source;
+}
+
+[RequireComponent(typeof(AudioSource))]
+public class GlobalMusic : MonoBehaviour
```

Comment:
What is the purpose of this class? How will it integrate with the Ink script? There is already a direction in the InkManager for playing music how does this relate?

Reply:
I didn't check, does the ink direction let you blend from one track to another?

Reply:
removed from current pr